lib/stages/pulsarPostProcess.cpp

```diff
@@ -304,11 +305,11 @@ void pulsarPostProcess::main_thread() {
                                             + (in_frame_location * _num_pol + p)
                                             + udp_pulsar_header_size;
                             } else if (_timesamples_per_pulsar_packet == 625) {
-                                // beam->packets->[time-freq-pol]
+                                // beam->packets->[freq-time-pol]
                                 out_index = psr * _udp_pulsar_packet_size * _num_packet_per_stream
                                             + frame * _udp_pulsar_packet_size
-                                            + (in_frame_location * _num_gpus * _num_pol
-                                               + thread_id * _num_pol + p)
+                                            + (thread_id * _samples_per_data_set * _num_pol
```

Comment:
Regardless of if you just revert to only making this index change or not, I'm fairly sure you need this change: 
```suggestion
                                            + (thread_id * _timesamples_per_pulsar_packet * _num_pol
```

Reply:
I agree, this should be `_timesamples_per_pulsar_packet` because we are just counting time within a packet.  `_samples_per_data_set` is a much larger number.

Reply:
thanks for catching this @andrerenard @cherryng, fixed in the latest commit.